src/components/AppLayout/Sidebar/SafeHeader/index.tsx

```diff
@@ -136,6 +209,11 @@ const SafeHeader = ({
 
   const hasSafeOpen = useRouteMatch(ADDRESSED_ROUTE)
 
+  const handleNewTransactionClick = () => {
+    trackEvent({ ...OVERVIEW_EVENTS.NEW_TRANSACTION })
+    onNewTransactionClick()
+  }
```

Comment:
Does the tracking component not work?

Reply:
Had to make the button take the full available width which didn't work with the div wrapper from `<Track>`